### ISSUE TITLE
No log when run on que CLI

### ISSUE THREAD

**Issue by drselump14** [que-rb/que#310]
I can't find any log when running the worker with que CLI

```
bundle exec que --log-level debug -q default --log-internals
Que waiting for jobs...
```

Adding a job works fine, and workers picked up the job but no information on the log.

Did I miss something?

This is my job 
```
module TradeContracts
  class ReviseWhenExpiredJob < ApplicationJob
    extend T::Sig
    queue_as :default

    sig { params(trade_contract: TradeContract).returns(T.untyped) }
    def perform(trade_contract:)
      return unless should_execute?(trade_contract)

      TradeContract::Operation::AutomaticRevise.call(model: trade_contract)
    end

    sig { params(trade_contract: TradeContract).returns(T::Boolean) }
    def should_execute?(trade_contract)
      trade_contract.may_revise? && trade_contract.expiration_job_id.present?
    end
  end
end
```
```
job = TradeContracts::ReviseWhenExpiredJob.set(wait_until: 30.seconds.from_now).perform_later(trade_contract: t)
```

```
# Gemfile
gem "que", "~> 1.0.0.beta4"
gem "que-scheduler"
gem "que-web"
```

The Que::Job format didn't work either.

**Comment by ZimbiX** [comment]
See the [logging section of the docs](https://github.com/que-rb/que/tree/master/docs#logging).

We use a logging middleware:

```ruby
# frozen_string_literal: true

require 'dex_provider_bridge/que/locks/advisory_lock'
require_relative 'serialised_execution_key'

module DexProviderBridge
  module Que
    module JobMiddleware
      class Logging
        def initialize(logger:)
          @logger = logger
        end

        def call(job)
          job_args = args(job)

          logger.info("Starting #{job.job_name} job", **job_args)

          before = Time.now.getutc

          yield

          duration = '%.2f' % (Time.now.getutc - before)

          logger.info("Finished #{job.job_name} job in #{duration}s", **job_args)
        end

        private

        def args(job)
          job.que_attrs[:args].reduce({}, :merge)
        end

        attr_reader :logger
      end
    end
  end
end
```

```ruby
middlewares = [
  DexProviderBridge::Que::JobMiddleware::Logging.new(logger: app['logger']),
  # ...
]
middlewares.each do |middleware|
  Que.job_middleware.push(middleware)
end
```

**Comment by oeoeaio** [comment]
Closing this now @drselump14, feel free to re-open if still an issue.

**Comment by pcantrell** [comment]
For anyone who's blocked waiting for a fix for this to upgrade Rails, here is a terrible hack that makes the error disappear. BEWARE: This may do terrible things; it's just a monkey patch that applies the fix in #423, and I have _no idea_ whether it's a good idea.

In `config/initializers/que.rb`:
```
# Temporary workaround for https://github.com/que-rb/que/issues/425
# until Que merges https://github.com/que-rb/que/pull/423 or similar
unless ::ActiveRecord::Base.respond_to?(:clear_active_connections!)
  class ::ActiveRecord::Base
    def self.clear_active_connections!
      ::ActiveRecord::Base.connection_handler.clear_active_connections!(:all)
    end
  end
end
```
